src/service/save-dispense.js

```diff
@@ -0,0 +1,27 @@
+import { postRequest } from '../utils/api-utils';
+
+// eslint-disable-next-line import/prefer-default-export
+export const saveDispense = async (data) => {
+  console.log('data', data)
```

Comment:
Can we remove this console log?

Reply:
this console.log was added for development purpose which is removed as part of "save dispense" card